components/feature/prompts/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Request code for picking a file from another activity -->
+    <item name="file_picker_activity_request_code" type="id" />
```

Comment:
We normally use a `mozac` prefix you can find an example [here](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/res/values/ids.xml)